caikit/core/data_model/base.py

```diff
@@ -82,176 +86,162 @@ def __new__(mcs, name, bases, attrs):
         _fields_map = ()
         proto_class = None
         full_name = name
-        if name != "DataBase":
-            # Look for the proto class. There are two places it could be:
```

Comment:
Getting rid of this logic is a big step towards removing reliance on project structure!